### ISSUE TITLE
Multivariate LSTM with Batch Normalization Clarification

### ISSUE THREAD

**Issue by whitmark** [Microsoft/CNTK#3319]
I am trying my hand at a multivariate LSTM but would also like to incorporate BatchNorm so the normalization coefficients can be retained in the model file.  The data I am working with has 7 time-steps in the sequence, 6 features, and 3 categorical labels as follows:

```
31 |F 2460.25 2469.00 2456.50 2465.50 1574076 54.28508 |L 1 0 0 |#  Bar: 32 Time: 9/6/2017 5:00:00 PM Ticks: 21
31 |F 2461.25 2471.25 2445.50 2459.75 2123649 54.15524
31 |F 2470.25 2479.75 2470.00 2474.25 1057024 56.53223
31 |F 2455.25 2474.25 2454.75 2470.00 1752826 53.17476
31 |F 2446.50 2459.75 2442.75 2455.75 1301444 48.24616
31 |F 2434.00 2448.50 2421.00 2447.00 1615262 45.13358
31 |F 2442.00 2449.25 2436.25 2443.75 953054 43.82434
32 |F 2462.25 2467.00 2455.75 2464.75 699851 54.14262 |L 0 1 0 |#  Bar: 33 Time: 9/7/2017 5:00:00 PM Ticks: 10
32 |F 2460.25 2469.00 2456.50 2465.50 1574076 54.28508
32 |F 2461.25 2471.25 2445.50 2459.75 2123649 54.15524
32 |F 2470.25 2479.75 2470.00 2474.25 1057024 56.53223
32 |F 2455.25 2474.25 2454.75 2470.00 1752826 53.17476
32 |F 2446.50 2459.75 2442.75 2455.75 1301444 48.24616
32 |F 2434.00 2448.50 2421.00 2447.00 1615262 45.13358
33 |F 2464.75 2465.75 2455.25 2461.00 1465772 53.00696 |L 0 0 1 |#  Bar: 34 Time: 9/8/2017 5:00:00 PM Ticks: -15
33 |F 2462.25 2467.00 2455.75 2464.75 699851 54.14262
33 |F 2460.25 2469.00 2456.50 2465.50 1574076 54.28508
33 |F 2461.25 2471.25 2445.50 2459.75 2123649 54.15524
33 |F 2470.25 2479.75 2470.00 2474.25 1057024 56.53223
33 |F 2455.25 2474.25 2454.75 2470.00 1752826 53.17476
33 |F 2446.50 2459.75 2442.75 2455.75 1301444 48.24616
```
I am starting with the simple sequence classification example provided [here](https://github.com/Microsoft/CNTK/blob/master/Examples/SequenceClassification/SimpleExample/Python/SequenceClassification.py) .

```
# Creates the reader
def create_reader(path, is_training, input_dim, label_dim):
    return C.io.MinibatchSource(C.io.CTFDeserializer(path, C.io.StreamDefs(
        features = C.io.StreamDef(field='x', shape=input_dim,   is_sparse=True),
        labels   = C.io.StreamDef(field='y', shape=label_dim,   is_sparse=False)
    )), randomize=is_training, max_sweeps = C.io.INFINITELY_REPEAT if is_training else 1)


# Defines the LSTM model for classifying sequences
def lstm_sequence_classifier(features, num_classes, embedding_dim, LSTM_dim):
    classifier = C.layers.Sequential([C.layers.Embedding(embedding_dim),
                                      C.layers.Recurrence(C.layers.LSTM(LSTM_dim)),
                                      C.sequence.last,
                                      C.layers.Dense(num_classes)])
    return classifier(features)

```

With no multivariate LSTM examples to reference, I am looking for some clarification on how to amend the simple example reader and LSTM definition to accommodate the multivariate nature of the inputs and include a BatchNormalization layer in place of the Embedding layer.

Thanks!

**Comment by KeDengMS** [comment]
Please take a look at the [ctf file](https://github.com/Microsoft/CNTK/blob/master/Tests/EndToEndTests/Text/SequenceClassification/Data/Train.ctf) used in the example.

In your case, the field name should be 'F' and 'L' respectively.
